packages/code-studio/src/main/AppControlsMenu.jsx

```diff
@@ -169,7 +169,7 @@ const AppControlsMenu = props => {
           onClearFilter();
         },
         order: 60,
-        shortcut: GRID_SHORTCUTS.TABLE.CLEAR_FILTERS,
+        shortcut: GRID_SHORTCUTS.TABLE.CLEAR_ALL_FILTERS,
```

Comment:
It bugs me that you named it `GRID_SHORTCUTS` here, but `IRIS_GRID_SHORTCUTS` in AppMainContainer. Would like it to be consistent.